### ISSUE TITLE
Password visibilty button is not tabbable

### ISSUE THREAD

**Issue by MelissaAutumn** [thunderbird/services-ui#250]
When you're tabbing through inputs you should be able to select the password visibility button and select it.

Additionally this should probably apply to the maxLength counter in some sort of way.
